DESCRIPTION

```diff
@@ -1,12 +1,22 @@
 Package: protocolhelper
 Title: Helper Functions to Manage Protocols
-Version: 0.0.0.9000
+Version: 0.1.0
```

Comment:
> For now, you could rewrite the branch after accepting/merging other proposed commits, in order to prevent successive commits from having 0.1.0. We can help if needed.


@florisvdh Any help is appreciated, or some guidance because I don't know what you want me to do.